plaso/parsers/locate.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+"""Parser for locate/updatedb database files."""
+
+import os
+
+from struct import unpack as struct_unpack
```

Comment:
Plaso uses dtfabric instead of stucts to make parsers easier to maintain

Reply:
Updated parser to use dtFabric and removed struct imports